src/views/mfa-verify/PushForm.js

```diff
@@ -62,7 +79,9 @@ define(['okta'], function (Okta) {
       }
     },
     submit: function (e) {
-      e.preventDefault();
```

Comment:
`(e) && (e.preventDefault())` will do the trick

or maybe directly call `_.defer(_.bind(this.doSave, this));` ?